src/luarocks/admin/cmd/add.lua

```diff
@@ -14,7 +14,7 @@ local index = require("luarocks.admin.index")
 function add.add_to_parser(parser)
    local cmd = parser:command("add", "Add a rock or rockspec to a rocks server.",
       util.see_also())
-      :add_help(false)
+      :add_help("--help")
```

Comment:
It this `--help` rather than `true` so that `-h` doesn't work?

Reply:
Yes, I could remove those lines if the default `-h --help` is good.